doc/releases/release-notes-2.1.rst

```diff
@@ -96,6 +96,11 @@ Build and Infrastructure
   Use new functions that utilize eDTS info such as dt_node_reg_addr.
   See :zephyr_file:`scripts/kconfig/kconfigfunctions.py` for details.
 
+* Deprecated direct use of the `DT_` Kconfig symbols from the generated
+  `generated_dts_board.conf`.  We should now utilize either functions from
```

Comment:
Could do `` :file:`generated_dts_board.conf` ``.

Reply:
Does that make sense since what would the link be to for a generated file?

Reply:
Only :zephyr_file: creates a link. :file: is just to format it like a path (though I don't know if we treat those any different from identifiers and stuff).